woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Thumbnails: Interactivity API directives

## What

Fixes woocommerce/woocommerce-blocks#10629

## Why



This PR allows the user to switch the main image by clicking on the Thumbnails. Currently, there isn't any animation. I will work on it in a dedicated PR.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Edit the Single Product Template.
2. Add the Product Gallery.
3. Save it.
4. Visit a product that has multiple images.
5. Click on a thumbnail image.
6. Ensure that the main image changes with the clicked one.


## Screenshots or screencast





https://github.com/woocommerce/woocommerce-blocks/assets/4463174/27dc9df2-6ddb-4b22-8494-ef19df85c4b7



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
